view/frontend/web/js/view/payment/method-renderer/omise-offsite-ocbcpao-method.js

```diff
@@ -23,9 +23,9 @@ define(
             code: 'omise_offsite_ocbcpao',
             restrictedToCurrencies: ['sgd'],
             logo: {
-                file: "images/payment_ocbc_pao.png",
-                width: "80",
-                height: "32",
+                file: "images/ocbc_pao.png",
```

Comment:
should we make the actual image 30x30 ?

Reply:
I just copied from our Omise-WooCommerce plugin.

Reply:
I try to resize 30px and it become blurred. So I updated with 60px.